### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by prakharagrawal10**
added card

### CODE REVIEW

**Comment by prakharagrawal10** [comment]
prakhars card

**Review by CandidDeer — CHANGES_REQUESTED** [review]
Please provide 3 useful resources. Thx.

File: index.html [hunk 1]
```diff
@@ -122,6 +122,36 @@ <h2>A project for first-time contributions</h2>
...
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+
+        <div class="card">
+          <p class="name">Prakhar</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://www.twitter.com/your_user_handle" target="_blank">Your handle</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/prakhar-agra-10" target="_blank">prakhar-agra-10</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:133]:
```suggestion
            <a href="https://github.com/prakharagrawal10" target="_blank">prakharagrawal10</a>
```
this is your correct github url

File: index.html [hunk 2]
```diff
@@ -122,6 +122,36 @@ <h2>A project for first-time contributions</h2>
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+
+        <div class="card">
+          <p class="name">Prakhar</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://www.twitter.com/your_user_handle" target="_blank">Your handle</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:131]: if you have an x / a twitter account please use your correct link and name

File: index.html [hunk 3]
```diff
@@ -122,6 +122,36 @@ <h2>A project for first-time contributions</h2>
...
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/prakhar-agra-10" target="_blank">prakhar-agra-10</a>
+          </p>
+          <p class="about">about you</p>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:135]: write something about you

File: index.html [hunk 4]
```diff
@@ -122,6 +122,36 @@ <h2>A project for first-time contributions</h2>
...
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="First Resource">StackOverflow</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:140]:
```suggestion
                <a href="https://github.com/prakharagrawal10" target="_blank" title="Stack Overflow">Stack Overflow</a>
```

File: index.html [hunk 5]
```diff
@@ -122,6 +122,36 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Second Resource">Free Code Camp</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:143]:
```suggestion
                <a href="https://www.freecodecamp.org/" target="_blank" title="freeCodeCamp">freeCodeCamp</a>
```

File: index.html [hunk 6]
```diff
@@ -122,6 +122,36 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Third resource">Chat GPT</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:146]:
```suggestion
                <a href="https://chat.openai.com/" target="_blank" title="ChatGPT">ChatGPT</a>
```

**Review by StackOverflowIsBetterThanAnyAI — COMMENTED** [review]
I added some suggestions to your code

File: index.html (hunk 2 above)
Comment (TenTraicion) [inline index.html:131]:
```suggestion
          <p class="name">Prakhar</p>
          <p class="contact">
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]

**Review by TenTraicion — APPROVED** [review]

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@prakharagrawal10, Congratulations on your PR Merge, and Thanks for your Contribution. You can check your card on our [Live Website](https://syknapse.github.io/Contribute-To-This-Project)! It may take 2-3 minutes to make your card visible on our website. 

Welcome to the Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Pull request merged**